tests/lib/schema_validation_test.py

```diff
@@ -706,7 +727,7 @@ def test_non_interger_route_table(self, default_data):
 class TestOVSBridgeLinkAggregation:
     def test_valid_link_aggregation_port(self, default_data):
         link_aggregation_port = {
-            OVSBridge.Port.NAME: "bond",
+            OVSBridge.Port.NAME: InterfaceType.BOND,
```

Comment:
This is incorrect.

Reply:
In case you wonder why it is incorrect, it is a name here and not a type.

Reply:
Thanks for pointing this out!